charts/resident-ui/templates/NOTES.txt

```diff
@@ -1,22 +0,0 @@
-1. Get the application URL by running these commands:
```

Comment:
why this was removed

Reply:
Lalith told its not much required we don't have this file in some modules